docs/rest_api_reference.md

```diff
@@ -0,0 +1,4 @@
+# REST API Reference
+
+Jellyfish uses OpenAPI for describing its REST API.
+You can find it [here](https://github.com/jellyfish-dev/jellyfish/blob/2a6082feff574b9909551cb512105cde95bec467/openapi.yaml).
```

Comment:
Maybe it would be better to link to the current master? Or maybe there's a way to link to the latest release?